src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -77,7 +77,7 @@
     &.selected {
       color: $color-gray-dark;
 
-      &.va-sidenav-item-label-duplicate {
+      &.va-sidenav-item-label-underlined {
         text-decoration: underline;
```

Comment:
You could also leverage the design system utility `vads-u-text-decoration--underline` to do this, but you're also developing a new module, so up to you

Reply:
Attempted to do this but looks like it was causing some other underline issues, so might keep it as is for now 😬

![image](https://user-images.githubusercontent.com/12773166/69357612-63899000-0c53-11ea-88a3-04a5142bd08e.png)